### ISSUE TITLE
Only works for nodes with a small number of minipools

### ISSUE THREAD

**Issue by chrismarino** [chrismarino/rocketreturns#8]
App uses free API key from Alchemy that are easily overrun. App throttled the requests, but will still error out when a node has 6 or  more minipools.

**Comment by htimsk** [comment]
Appears to time out after 25 minipools.

**Comment by chrismarino** [comment]
Right. I was able to get things to work beyond 5 or 6 minipools, but still has probs above 20 or so. There is a lib that loops through minipools too quickly/async and haven't found a simple way to work around it.
